ctdcal/__main__.py

```diff
@@ -0,0 +1,34 @@
+import click
+from pathlib import Path
+
+
+@click.group()
+def cli():
+    """The ctdcal command creates and manipulates data directories
+
+    Documentation: tbd
+    """
+    pass
+
+
+@cli.command()
+def init():
+    """Setup data folder with appropriate subfolders"""
+
+    DEFAULT_DIRS = [
```

Comment:
If you're still using time, btl, pressure directories you might as well build them now, unless you plan to separate the incoming data from the intermediate data products.

Reply:
They get created by one of early functions from `process_all` (maybe `odf_sbe_metadata.py`) and I haven't yet bothered figuring out where that happens to remove redundancy.

On the flip side, I kind of like the idea of having a separation of directories the user should/shouldn't add to or edit.